test/jdk/java/awt/Frame/SetMinimumSizeTest/SetMinimumSizeTest1.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @key headful
+ * @summary Verify that increase in Frame's minimumSize gets reflected in the subsequent getSize call
+ * @run main SetMinimumSizeTest1
+ */
+
+import java.awt.Button;
+import java.awt.Dimension;
+import java.awt.EventQueue;
+import java.awt.Frame;
+import java.awt.Robot;
+
+public class SetMinimumSizeTest1 {
+
+    private static Frame frame;
+    private static volatile Dimension dimension, actualDimension;
+
+    public static void createGUI(Frame _frame) {
+        frame = new Frame();
+        frame.add(new Button("Button"));
+        frame.setSize(140, 140);
+        frame.setLocationRelativeTo(null);
+        frame.setVisible(true);
+    }
+
+    public static void doTest() throws Exception {
+        try {
+            EventQueue.invokeAndWait(() -> createGUI(frame));
+
+            Robot robot = new Robot();
+            robot.setAutoDelay(100);
+            robot.waitForIdle();
+
+            EventQueue.invokeAndWait(() -> {
+                dimension = frame.getSize();
+                dimension.width = dimension.width + 20;
+                dimension.height = dimension.height + 20;
+                frame.setMinimumSize(dimension);
+                frame.invalidate();
+                frame.validate();
+            });
+
+            robot.waitForIdle();
+
+            EventQueue.invokeAndWait(() -> {
+                actualDimension = frame.getSize();
+            });
+
+            robot.waitForIdle();
+
+            if (!actualDimension.equals(dimension)) {
+                throw new RuntimeException("Test Failed\n"
+                    + "expected dimension:(" + dimension.width + "," + dimension.height +")\n"
+                    + "actual dimension:(" + actualDimension.width + "," + actualDimension.height + ")");
+            }
+        }
+        finally {
```

Comment:
```suggestion
        } finally {
```

The finally-block as well as catch- and else-blocks should be at the same line as the closing brace from the previous block.